src/app/models/harvest/HarvestItemInfo.ts

```diff
@@ -0,0 +1,117 @@
+import { Injector } from "@angular/core";
+import { ACCOUNT, SHALLOW_SITE } from "@baw-api/ServiceTokens";
+import { DateTimeTimezone, Id } from "@interfaces/apiInterfaces";
+import { AbstractModelWithoutId } from "@models/AbstractModel";
+import { hasOne } from "@models/AssociationDecorators";
+import { bawDateTime, bawDuration } from "@models/AttributeDecorators";
+import { Site } from "@models/Site";
+import { User } from "@models/User";
+import { Duration } from "luxon";
+
+export interface IHarvestItemInfoFixes {
+  file?: string;
+  problems?: {
+    [key: string]: {
+      data?: null;
+      status?: string;
+      message?: string;
+      severity?: string;
+    };
+  };
+}
+
+export interface IHarvestItemInfoFileInfoNotes {
+  relativePath?: string;
+}
+
+export interface IHarvestItemInfoValidation {
+  name?: string;
+  status?: "fixable" | "notFixable";
+  message?: string;
+}
```

Comment:
Of this file, keep this definition.

The rest of it I am pretty sure I don't want to serialize.